contracts/lib/AssetAllowList.sol

```diff
@@ -0,0 +1,151 @@
+/*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    SPDX-License-Identifier: Apache License, Version 2.0
+*/
+
+pragma solidity 0.6.10;
+import { AddressArrayUtils } from "./AddressArrayUtils.sol";
+
+/**
+ * @title ModuleBase
+ * @author Set Protocol
```

Comment:
Let's patch copy pasta and add unit tests for this contract

Reply:
added here [`9175f17`](https://github.com/IndexCoop/index-coop-smart-contracts/pull/157/commits/9175f176bdcfe60d87bff9ed24d047d9f1f28056)